docs/data/material/experimental-api/css-variables/css-variables.md

```diff
@@ -41,7 +41,7 @@ To customize it using CSS variables, you'll need to wrap your application with `
 
 Play around with the demo below!
 
-{{"demo": "CssVariablesCustomization.js", "iframe": true }}
+{{"demo": "CssVariablesCustomization.js", "iframe": true, "iframeId": "css-vars-customization", "height": "100px" }}
```

Comment:
The reason why I used iframe here is because the change of the primary color in one demo reflected in all demos. Maybe instead of setting ids for the iframes, we should have just add the variables for that demo in a parent container and used it's id instead of in `:root` in just that specific demo.

I think we should spend more time looking into this, because long term, when we move to CSS variables, we would potentially hit this problem, and I don't think using iframes in each demo are the solution for it ultimately.

Reply:
> I think we should spend more time looking into this

Agree, but I think we don't have to change the iframe right?